lib/util.js

```diff
@@ -345,11 +349,35 @@ function getSystemErrorName(err) {
   return internalErrorName(err);
 }
 
+function addSafeAbortSignalAbortListener(signal, listener, options = kEmptyObject) {
```

Comment:
I don't think we need to provide options here?

Reply:
As in, the signature can just be `(signal, listener)`

Reply:
why not allow passing `capture` & `signal`?

Reply:
Why allow it? I don't see the point I want to abstract the event listener'ness away from it.